README.md

```diff
@@ -65,4 +64,4 @@ yo swill-boilerplate
 
 ## License
 
-Swill Boilerplate Generator is released under the terms of the [MIT license](https://github.com/tiagoporto/generator-swill-boilerplate/blob/master/LICENSE).
+Swill Boilerplate Generator is released under the terms of [MIT license](https://github.com/tiagoporto/generator-swill-boilerplate/blob/master/LICENSE).
```

Comment:
Look this https://help.github.com/articles/open-source-licensing/#where-does-the-license-live-on-my-repository